app/views/users/new.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Registration</h1>
+
+<%= render :partial => "form", locals: { path: "/users", method: :post, button_text: "Sign-up" } %>
```

Comment:
Glad we figured out this isn't working when we have an enter after the curly bracket (though @leahriffell said hers did work so this might be a good theory to research)